client/src/main/java/com/paypal/selion/internal/reports/runtimereport/MethodInfo.java

```diff
@@ -52,7 +52,7 @@
     private String exception;
     private String stacktrace;
     private List<LogInfo> logs;
-    private transient ITestResult result;
+    private final transient ITestResult result;
```

Comment:
sure. It does look odd at first glance.. I'll switch this to the gson annotation